src/cpp/cpp_typecheck_expr.cpp

```diff
@@ -1182,7 +1182,7 @@ void cpp_typecheckt::typecheck_expr_member(
       else
       {
         // it must be a static component
-        const struct_typet::componentt pcomp=
+        const struct_typet::componentt &pcomp =
           type.get_component(to_symbol_expr(symbol_expr).get_identifier());
```

Comment:
Seems unrelated to PR?

Reply:
I may have overstretched the "cleanup API use."